olivia_maina/for_loop_basic2.py

```diff
@@ -0,0 +1,96 @@
+# Biggie Size - Given an array, write a function that changes all positive numbers in the array to 
+# "big". Example: makeItBig([-1, 3, 5, -5]) returns that same array, changed to [-1, "big", "big", -5].
+def biggie(arr):
+    for i in range(len(arr)):
+        if arr[i] > 0:
+            arr[i] = 'Big'
+    return arr
+print(biggie([-1,3,5,-5]))
+
+#Count Positives - Given an array of numbers, create a function to replace last value with number of positive values. Example, 
+# countPositives([-1,1,1,1]) changes array to [-1,1,1,3] and returns it.  (Note that zero is not considered to be a positive number).
+def positives(arr):
+    total = 0
+    for i in range(len(arr)):
+        if arr[i]>0:
+            total += 1
+    arr[len(arr)-1] = total
+    return arr 
+print(positives([-1,1,1,1]))
+
+#SumTotal - Create a function that takes an array as an argument and returns the sum of all the values in the array.  
+# For example sumTotal([1,2,3,4]) should return 10
+def sumtotal(arr):
+    sum=0
+    for i in range(len(arr)):
+        sum += arr[i]
+    return sum
+print(sumtotal([1,2,3,4]))
+
+#Average - Create a function that takes an array as an argument and returns the average of all the values in the
+#  array.  For example multiples([1,2,3,4]) should return 2.5
+
+def average(arr):
+    avg=0
+    for i in range(len(arr)):
+        avg += arr[i]
+    return avg/len(arr)
+print(average([1,2,3,4]))
+
+#Length - Create a function that takes an array as an argument and returns the length of the array. 
+#  For example length([1,2,3,4]) should return 4
+def length(arr):
+    return len(arr)
```

Comment:
Another way: you can also count the elements in the array. Initialize count to 0... for i in arr....increment count and return the final count at the end.